interface/themes/core.scss

```diff
@@ -12,9 +12,6 @@
 @import "../../public/assets/@ttskch/select2-bootstrap4-theme/src/layout";
 
 @import "directional";
-
-@import url("themeBuilder.php"); // reads from user preferences and overrides fonts
```

Comment:
whooohooooo, to the fire! :)

![tenor (4)](https://user-images.githubusercontent.com/278968/90206811-74f26980-dd99-11ea-83ba-0adce3530962.gif)